src/main/java/com/solr/clientwrapper/rest/EmployeeController.java

```diff
@@ -66,6 +73,76 @@ public Employee getEmployeeByAddress(@PathVariable String address) {
     private SolrClient solrClient;
     private final static String solrDataName = "employee";
 
+    @GetMapping("/coreStatus/{name}")
+    public String coreStatus(@PathVariable String name) throws SolrServerException, IOException, URISyntaxException, ParserConfigurationException, InterruptedException, TransformerException, org.xml.sax.SAXException {
+
+        HttpSolrClient solrClient = new HttpSolrClient.Builder("http://localhost:8983/solr").build();
+        CoreAdminResponse coreAdminResponse=CoreAdminRequest.getStatus(name,solrClient);
+
+        return coreAdminResponse.toString();
+    }
+
+    @GetMapping("/reloadCore/{name}")
+    public boolean reloadCore(@PathVariable String name) throws SolrServerException, IOException, URISyntaxException, ParserConfigurationException, InterruptedException, TransformerException, org.xml.sax.SAXException {
+
+        HttpSolrClient solrClient = new HttpSolrClient.Builder("http://localhost:8983/solr").build();
+
+        CoreAdminResponse coreAdminResponse=CoreAdminRequest.reloadCore(name,solrClient);
+
+        System.out.println(coreAdminResponse);
```

Comment:
replace sysout with logger

Reply:
Employee Controller was only for testing. Removed it and also the other related classes.